kernel/packages/build-ecs/index.ts

```diff
@@ -162,11 +162,11 @@ async function minify(files: string | string[] | { [file: string]: string }) {
     compress: {
       passes: 2
     },
-    format: {
-      ecma: 5,
-      comments: /^!/,
-      beautify: false
-    },
+    // format: {
+    //   ecma: 5,
+    //   comments: /^!/,
+    //   beautify: false
+    // },
```

Comment:
Please restore this, it is necessary

Reply:
yes!, of course. I don't know why these lines are comment, maybe @2fd knows